js/main.js

```diff
@@ -46,7 +46,7 @@ if (shouldQuit) {
 
 var symphonyAutoLauncher = new AutoLaunch({
     name: 'Symphony',
-    path: process.execPath,
+    path: isMac ? '/Applications/' + app.getName() + '.app' : process.execPath,
```

Comment:
By doing that i was facing this [issue](https://github.com/Teamwork/node-auto-launch/issues/28). Auto launcher verifies the path and if it fails to locate, it will defaults to the ```exe`` path.